hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/validation/StandardizedAttributeValidator.java

```diff
@@ -136,4 +137,11 @@ private static boolean contains(byte[] a, byte val) {
         }
         return false;
     }
+
+    @Override
+    public boolean isImmutableKey(@NonNull Key key) {
```

Comment:
This method name seems very unintuitive, since “immutable” describes the key, but in this case doesn’t describe the key, instead describing the immutable entity associated with it. A comment about what this actually means would be helpful. Even better would be to use a name that doesn't label the key "immutable" (when the key isn't actually immutable)

Reply:
There is a Java doc in interface AttributeValidator: Validates if immutable entity with the key and @return true if immutable entity with the key